ios/RNTPTDocumentView.m

```diff
@@ -6221,6 +6221,45 @@ - (void)setSignatureColors:(NSArray *)signatureColors
     [self applyViewerSettings];
 }
 
+- (NSDictionary *)addAnnotation:(NSString *)type fieldName:(NSString *)fieldName pageNumber:(NSInteger *)pageNumber x1:(NSNumber * _Nonnull)x1 y1:(NSNumber * _Nonnull)y1 x2:(NSNumber * _Nonnull)x2 y2:(NSNumber * _Nonnull)y2;
```

Comment:
@sryorch Suggestions for your refactoring. 

- Remove the return value NSDictionary.
- The x/y parameters could be of type `double` instead.
- Delete fieldMap